src/index.html

```diff
@@ -18,12 +18,32 @@
             integrity="sha384-vp86vTRFVJgpjF9jiIGPEEqYqlDwgyBgEF109VFjmqGmIY/Y4HV4d3Gp2irVfcrp"
             crossorigin="anonymous"
         />
+        <!-- CSS -->
+        <link rel="stylesheet" href="./style.css" />
         <title></title>
     </head>
 
     <body>
+        <div class="wrapper">
+            <div class="wrapper--transparency"></div>
+            <header></header>
+            <main id="main" class="container-lg">
```

Comment:
we are going to inject main content dynamically. Each view will be injected here (probably here) so when user switches to another view your content will be removed and never dislayed again